internal/tui/tui.go

```diff
@@ -127,15 +129,16 @@ func Start(manual bool, data *Data) {
 	tick(s, data, manual, quit)
 	s.Fini()
 
-	log.Infof("Total cost: $%.2f", data.getCost())
+	log.Infof("Total cost: %s%.2f", data.currencySymbol, data.getCost())
 }
 
 // data stores variables passed around between the various goRoutines.
 type Data struct {
-	mtx   sync.Mutex
-	count int
-	cost  float32
-	input string
+	mtx            sync.Mutex
+	count          int
+	cost           float32
+	input          string
+	currencySymbol string
```

Comment:
I recommend against setting the currency Symbol in `data`. It's specifically designed for storing values that are frequently updated as the program executes and is intended to facilitate inter-process communication.

I recommend passing it to the [draw function](https://github.com/syncfast/clockwise/blob/42861ee9f4c888605a4e3c1e85d1c0f61eb93d93/internal/tui/tui.go#L221) as a standalone parameter.

Something like:
```
func draw(s tcell.Screen, data *Data, manual bool, currency string) {
``` 

`Start` is making calls to viper to retrieve the currency symbol anyway, so it can inject the value when it calls `draw`.

Reply:
Alternatively, thinking ahead, we can setup a `config` struct that includes fields like `manual` and `currencySymbol` and potentially other config values, then just pass that to draw.

Reply:
Otherwise looks great. Really appreciate the contribution and your patience.

Reply:
Thinking out loud here: viper might be storing the config in memory, in which case, using viper to retrieve the value within draw might have no penalties vis a vis passing in our own param or config struct.

Reply:
Config struct sounds nice imo, cause it can also hold values like the following in the future

```go
virtual_webcam_name string
cost_report_filepath string
...
```

so Config struct it is, rather than distinct `currency string` parameter to draw?

Reply:
Create struct `Config` in package `tui`. Lemme hear your feedback ;D